packages/editor/src/client/utility.ts

```diff
@@ -247,3 +247,23 @@ export function isValueConstructor<T>(value: T | (() => T)): value is () => T {
 export function isFunctionalUpdate<T>(value: React.SetStateAction<T>): value is (value: T) => T {
   return typeof value === 'function'
 }
+
+// replace scripts with executable ones
+export function executeScriptElements(containerElements: NodeListOf<HTMLElement> | HTMLElement[]) {
+  if (!containerElements.length) return
+
+  Array.from(containerElements).forEach(container => {
```

Comment:
Why do we need exactly three inner forEach() ? Looks a bit shady. Can we resolve that somehow or program it more generic?